lib/main/widgets/pet_list_item.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class PetListItem extends StatelessWidget {
+  final String namePet;
+  final String breedPet;
+
+  //final Image imagePet;
```

Comment:
I think you should add the image functionality because this widget is not done yet